### ISSUE TITLE
Blank page on Safari and playlists not working 

### ISSUE THREAD

**Issue by Darkcrown96** [swing-opensource/swingmusic#155]
Hi, i recently installed the v1.3.0 through docker-compose and i'm having some issues. On Safari (i've tried on my iPhone, iPad and Macbook) appears only a white blank page, even in PWA while on my windows machine works fine expect for the playlists. 
I created some of them just for testing but nothing, they won't start. 
The app is behind nginx proxy manager.

What i have to do?

**Comment by mungai-njoroge** [comment]
Hello @Darkcrown96 

Thanks for opening this issue. I haven't tested the app on Safari, so I don't know what's going on. Have you tried other browsers? I had a blank page issue on older releases on Windows, which was being caused by wrong mime types (It was fixed on `v1.3.0`). Can you check whether there are errors in the console. A screenshot would be nice.

Have you tried loading the page without the proxy?

About the Windows playlist issue, can you share a screenshot?

**Comment by Darkcrown96** [comment]
I tried with Chrome and Edge and everything is fine. On Safari i have the blank page even without proxy. But, with the proxy I receive the errors in the screenshot, without I receive only the first error.
![photo_2023-10-28_20-01-10](https://github.com/swing-opensource/swingmusic/assets/126989751/31c60337-2f40-4ab1-aac3-700833002373)

**Comment by mungai-njoroge** [comment]
Oh snap!

It looks like Safari does not accept compressed files. Fixing for Safari will break page for Chrome and others.

**Comment by Darkcrown96** [comment]
Will be a fix for the next release?
For the playlists i can't do a screenshot, maybe a video, because is the action of start playing that doesn't work.

**Comment by mungai-njoroge** [comment]
@Darkcrown96 

I can do a build an hot fix build for you and push it to Docker hub if you want.

Open the console, try to play and check if any errors occur. If any, take a screenshot and paste them in your reply.

**Comment by Darkcrown96** [comment]
Would be nice, for sure. And maybe would help someone else in the same situation.
But i don't wanna bother you.

**Comment by mungai-njoroge** [comment]
@Darkcrown96 

I've removed the compression completely in the patch build. Try pulling the `v1.3.1.safari_fix` version and see whether it works. I'll try to patch it properly in a future release.

```sh
docker pull ghcr.io/swing-opensource/swingmusic:v1.3.1.safari_fix
```

[`v1.3.1.safari_fix` on Github Registry](https://github.com/swing-opensource/swingmusic/pkgs/container/swingmusic/142165335?tag=v1.3.1.safari_fix)

Also ... I'd appreciate if you provided the screenshots or logs for the playlist playing issue.

Thanks ... and enjoy!

**Comment by Darkcrown96** [comment]
I'll try and let u know. For the playlist issue, here you are.
![Cattura](https://github.com/swing-opensource/swingmusic/assets/126989751/40aedce6-7ff2-4337-ad54-8e1091b38b3a)

**Comment by mungai-njoroge** [comment]
Hello @Darkcrown96 

This issues has been fixed by #157. Pull the latest release to get the new features and bug fixes.
